plugins/sonarqube/package.json

```diff
@@ -47,7 +47,8 @@
     "@material-ui/styles": "^4.10.0",
     "cross-fetch": "^3.1.5",
     "rc-progress": "3.4.1",
-    "react-use": "^17.2.4"
+    "react-use": "^17.2.4",
+    "luxon": "^3.3.0"
```

Comment:
We use `^3.0.0` in the rest of the project, let's keep it uniform. And you'll need to run `yarn install` too and commit the `yarn.lock` change in your PR

Reply:
My bad vscode nudged me